### PR TITLE
Add two new languages English and Spanish

### PR DESCRIPTION
**Pull request by jeann2013**
**Describe Pull request**
We have added English and Spanish language for this resource.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -107,12 +107,12 @@ CreateThread(function()
                         ClosestTarget = k
                         if plantData["plantStats"]["health"] > 0 then
                             if plantData["plantStage"] ~= plantData["plantStats"]["highestStage"] then
-                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"], 'Sort: '..plantData["plantSort"]["label"]..'~w~ ['..plantData["plantStats"]["gender"]..'] | Nutrition: ~b~'..plantData["plantStats"]["food"]..'% ~w~ | Health: ~b~'..plantData["plantStats"]["health"]..'%')
+                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"], ''.. Lang:t('text.sort') ..''.. plantData["plantSort"]["label"]..'~w~ ['..plantData["plantStats"]["gender"]..'] | '..Lang:t('text.nutrition')..' ~b~'..plantData["plantStats"]["food"]..'% ~w~ | '..Lang:t('text.health')..' ~b~'..plantData["plantStats"]["health"]..'%')
```
Comment (tom-osborne) [inline client/main.lua]: No need to concatenate empty strings here

File: client/main.lua [hunk 2]
```diff
@@ -107,12 +107,12 @@ CreateThread(function()
...
                             else
-                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"] + 0.2, "Press ~g~ E ~w~ to harvest plant.")
-                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"], 'Sort: ~g~'..plantData["plantSort"]["label"]..'~w~ ['..plantData["plantStats"]["gender"]..'] | Nutrition: ~b~'..plantData["plantStats"]["food"]..'% ~w~ | Health: ~b~'..plantData["plantStats"]["health"]..'%')
+                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"] + 0.2, Lang:t('text.harvest_plant'))
+                                DrawText3Ds(plantData["plantCoords"]["x"], plantData["plantCoords"]["y"], plantData["plantCoords"]["z"], ''..Lang:t('text.sort')..' ~g~'..plantData["plantSort"]["label"]..'~w~ ['..plantData["plantStats"]["gender"]..'] | '..Lang:t('text.nutrition')..' ~b~'..plantData["plantStats"]["food"]..'% ~w~ | '..Lang:t('text.health')..' ~b~'..plantData["plantStats"]["health"]..'%')
```
Comment (tom-osborne) [inline client/main.lua]: Again, no need to concatenate empty strings here

File: fxmanifest.lua [hunk 3]
```diff
@@ -4,13 +4,17 @@ game 'gta5'
 description 'QB-Weed'
 version '1.0.0'
 
-shared_script 'config.lua'
-
-client_script 'client/main.lua'
+shared_script {
```
Comment (tom-osborne) [inline fxmanifest.lua]: This should be plural "shared_scripts"
```lua
shared_scripts {
```

File: server/main.lua [hunk 4]
```diff
@@ -155,19 +155,19 @@ RegisterNetEvent('qb-weed:server:harvestPlant', function(house, amount, plantNam
...
-                    TriggerClientEvent('QBCore:Notify', src, 'This plant no longer exists?', 'error', 3500)
+                    TriggerClientEvent('QBCore:Notify', src, Lang:t('error.this_plant_no_longer_exists') , 'error', 3500)
                 end
             else
-                TriggerClientEvent('QBCore:Notify', src, 'House Not Found', 'error', 3500)
+                TriggerClientEvent('QBCore:Notify', src, Lang:t('error.house_not_found'), 'error', 3500)
             end
         else
-            TriggerClientEvent('QBCore:Notify', src, "You Don't Have Enough Resealable Bags", 'error', 3500)
+            TriggerClientEvent('QBCore:Notify', src, Lang:t('error.you_dont_have_enough_resealable_bags') , 'error', 3500)
```
Comment (tom-osborne) [inline server/main.lua]: Remove whitespace before comma

File: server/main.lua [hunk 5]
```diff
@@ -155,19 +155,19 @@ RegisterNetEvent('qb-weed:server:harvestPlant', function(house, amount, plantNam
                     Player.Functions.RemoveItem('empty_weed_bag', sndAmount)
                     MySQL.query('DELETE FROM house_plants WHERE plantid = ? AND building = ?',
                         {plantId, house})
-                    TriggerClientEvent('QBCore:Notify', src, 'The plant has been harvested', 'success', 3500)
+                    TriggerClientEvent('QBCore:Notify', src,  Lang:t('text.the_plant_has_been_harvested'), 'success', 3500)
                     TriggerClientEvent('qb-weed:client:refreshHousePlants', -1, house)
                 else
-                    TriggerClientEvent('QBCore:Notify', src, 'This plant no longer exists?', 'error', 3500)
+                    TriggerClientEvent('QBCore:Notify', src, Lang:t('error.this_plant_no_longer_exists') , 'error', 3500)
```
Comment (tom-osborne) [inline server/main.lua]: Remove whitespace before comma

File: server/main.lua [hunk 6]
```diff
@@ -155,19 +155,19 @@ RegisterNetEvent('qb-weed:server:harvestPlant', function(house, amount, plantNam
                     Player.Functions.RemoveItem('empty_weed_bag', sndAmount)
                     MySQL.query('DELETE FROM house_plants WHERE plantid = ? AND building = ?',
                         {plantId, house})
-                    TriggerClientEvent('QBCore:Notify', src, 'The plant has been harvested', 'success', 3500)
+                    TriggerClientEvent('QBCore:Notify', src,  Lang:t('text.the_plant_has_been_harvested'), 'success', 3500)
```
Comment (tom-osborne) [inline server/main.lua:158]: Remove extra whitespace before `Lang`

**Review by tom-osborne — CHANGES_REQUESTED** [review]
@jeann2013 Some small changes please.

File: client/main.lua (hunk 1 above)
Comment (jeann2013) [reply client/main.lua]: @tom-osborne Done

File: client/main.lua (hunk 2 above)
Comment (jeann2013) [reply client/main.lua]: @tom-osborne Done

File: fxmanifest.lua (hunk 3 above)
Comment (jeann2013) [reply fxmanifest.lua]: @tom-osborne Done

File: server/main.lua (hunk 4 above)
Comment (jeann2013) [reply server/main.lua]: Done

File: server/main.lua (hunk 5 above)
Comment (jeann2013) [reply server/main.lua]: Done

File: server/main.lua (hunk 6 above)
Comment (jeann2013) [reply server/main.lua:158]: Done

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**